test_root/semgrep/README.rst

```diff
@@ -0,0 +1,21 @@
+Semgrep linters
+###############
+
+Linting rules for use with `semgrep`_ during CI checks on PRs.
+
+Status
+******
+
+This is an experimental approach to developing new linting rules. Semgrep provides by-example structural matching that can be easier to write and maintain than procedural code inspecting ASTs. If the approach works out, we can expand our use of Semgrep; if it becomes a problem for some reason, we can switch to adding pylint rules in edx-lint.
+
+How-to
+******
+
+If you need to tell semgrep to ignore a block of code, put a ``# nosemgrep`` comment on or before the first matched line.
```

Comment:
Could you either add a subtitle like `Ignoring failures` (or something of that nature), or change `How-to`? I was going to ask you to add this very note that already existed, because it didn't stand out enough for me.

Reply:
Yeah, I can make it more specific.